src/main/webapp/app/core/login/login.service.ts

```diff
@@ -9,6 +9,8 @@ import { AccountService } from 'app/core/auth/account.service';
 import { JhiWebsocketService } from 'app/core/websocket/websocket.service';
 import { NotificationService } from 'app/shared/notification/notification.service';
 
+export type Saml2 = { ['button-label']: string; ['enable-password']: boolean };
```

Comment:
This is still no class. Please use a proper class in its own file and call if Saml2Config